src/app/app/onboarding/_components/OnboardingComplete.tsx

```
@@ -0,0 +1,105 @@
+import {
+  IconCalendarEvent,
+  IconCamera,
+  IconCircleCheck,
+  IconSearch,
+} from "@tabler/icons-react";
+
+const FIRST_ACTIONS = [
+  {
+    icon: IconCamera,
+    title: "Upload portfolio photos",
+    description: "Show leads what you can do",
```

ESLint

[error] Expected object keys to be in natural insensitive ascending order. 'description' should be before 'title'.

(sort-keys-fix/sort-keys-fix)